posthog-react-native/src/storage.ts

```diff
@@ -1,69 +1,69 @@
-import * as FileSystem from 'expo-file-system'
+import { PostHogCustomAsyncStorage } from './types'
 
 const POSTHOG_STORAGE_KEY = '.posthog-rn.json'
 const POSTHOG_STORAGE_VERSION = 'v1'
 
 type PostHogStorageContents = { [key: string]: any }
 
-const loadStorageAsync = async (): Promise<PostHogStorageContents> => {
-  const uri = (FileSystem.documentDirectory || '') + POSTHOG_STORAGE_KEY
-  // If we change POSTHOG_STORAGE_VERSION then we should migrate the persisted data here
-  try {
-    const stringContent = await FileSystem.readAsStringAsync(uri)
-    return JSON.parse(stringContent).content
-  } catch (e) {
-    return {}
-  }
-}
+// NOTE: The core prefers a synchronous storage so we mimic this by pre-loading all keys
```

Comment:
flyby nit: "a synchronous" or "asynchronous"
```suggestion
// NOTE: The core prefers asynchronous storage so we mimic this by pre-loading all keys
```

Reply:
haha I commited this and realised its incorrect. It's meant to be "a synchronous" (as in not-async).

Reply:
Is it better then as "...the core prefers synchronous..." so it's more parseable

This is, of course, the most important thing to focus on in this PR :p